app/models/concerns/paginate.rb

```diff
@@ -0,0 +1,17 @@
+module Paginate
+  def paginate(limit_query, page_query)
+    limit(paginate_limit(limit_query)).order("id ASC").
```

Comment:
Pagination bug on production environment fixed by explicitly ordering the query.